### PR TITLE
Removing unused array libraries, adding some comments, & minor cleanup

### PR DESCRIPTION
**Pull request by adrake33**

### CODE REVIEW

File: tests/libraries/test_arrays.py [hunk 1]
```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env python
```
Comment (nuevoalex) [inline tests/libraries/test_arrays.py]: Why do we delete these tests? They're testing the uint256 array library which is still being used I think

**Comment by nuevoalex** [comment]
Look good but I think we should keep the array library test and test helper

File: tests/libraries/test_arrays.py (hunk 1 above)
Comment (adrake33) [reply tests/libraries/test_arrays.py]: What's using the uint256 array library?  The only place I saw it being used was ArrayHelper, which was only being used by test_arrays.py.

Comment (nuevoalex) [reply tests/libraries/test_arrays.py]: not used I guess, just not deleted. As long as we're keeping it I'd prefer it stay tested.

**Comment by nuevoalex** [comment]
Approving. I think either delete the uint256 array library too though or keep the tests. Either is fine with me

**Comment by adrake33** [comment]
Ah, sorry about that.  I meant to delete the uint256 array library as well, but it looks like I missed it when I was telling Git to remove the other array libraries.  I'll remove it.

**Pull request merged**